builds/jazz-build-module/sls-app/sbr.groovy

```diff
@@ -4,15 +4,18 @@ import org.json.*
 
 @Field def whitelistValidator
 @Field def sbrContent
+@Field def whiteListYml
 
 echo "sbr.groovy is being loaded"
 
 def output
 
-def initialize(output,aWhitelistValidator) {
+def initialize(output, aWhitelistValidator) {
   this.output = output
   sbrContent = readFile("./sls-app/serverless-build-rules.yml")
   whitelistValidator = aWhitelistValidator
+  whitelistContent = readFile("./sls-app/whitelist.yml")
```

Comment:
Since we already have the whitelistValidator which loads this. Can we have a method in the module to `whiteListYml['assetTypes']`?

Reply:
Implemented the suggestion